apps/snitch_core/priv/repo/migrations/20181212072305_add_url_to_image.exs

```diff
@@ -0,0 +1,9 @@
+defmodule Snitch.Repo.Migrations.AddUrlToImage do
+  use Ecto.Migration
+
+  def change do
+    alter table("snitch_images") do
+      add(:image_url, :string)
```

Comment:
@jyotigautam I recall we decided to rename the existing `image` field to `image_url`. correct me if I am wrong.